src/pages/FeaturePreviews.js

```diff
@@ -0,0 +1,90 @@
+import _ from 'lodash/fp'
+import { Fragment, useCallback, useEffect, useState } from 'react'
+import { div, h, h2, p, span } from 'react-hyperscript-helpers'
+import { Checkbox, PageBox, spinnerOverlay } from 'src/components/common'
+import FooterWrapper from 'src/components/FooterWrapper'
+import { SimpleFlexTable } from 'src/components/table'
+import TopBar from 'src/components/TopBar'
+import { isFeaturePreviewEnabled, toggleFeaturePreview, useAvailableFeaturePreviews } from 'src/libs/feature-previews'
+import * as Style from 'src/libs/style'
+import * as Utils from 'src/libs/utils'
+
+
+export const FeaturePreviews = () => {
+  const { featurePreviews, loading, error } = useAvailableFeaturePreviews()
+
+  // The source of truth for whether or not a feature preview is enabled is `isFeaturePreviewEnabled`, which reads
+  // from local preferences. However, changing a local preference won't re-render this component and its checkboxes.
+  // Thus, we also need to store this in component state and keep in sync with local preferences.
+  const getFeaturePreviewState = useCallback(() => {
+    return _.flow(_.map(({ id }) => [id, isFeaturePreviewEnabled(id)]), _.fromPairs)(featurePreviews)
+  }, [featurePreviews])
+
+  // Map of feature preview ID => enabled boolean
+  const [featurePreviewState, setFeaturePreviewState] = useState(getFeaturePreviewState)
+  useEffect(() => { setFeaturePreviewState(getFeaturePreviewState()) }, [getFeaturePreviewState])
+
+  return Utils.cond(
+    [loading, () => spinnerOverlay],
+    [error, () => p({ style: { margin: 0 } }, ['Unable to load feature previews.'])],
+    [_.size(featurePreviews) === 0, () => p({ style: { margin: 0 } }, ['No feature previews available at this time.'])],
```

Comment:
[Nit] 
```suggestion
    [_.isEmpty(featurePreviews), () => p({ style: { margin: 0 } }, ['No feature previews available at this time.'])],
```
might be slightly simpler.